exercises/complex-numbers/canonical-data.json

```
@@ -363,6 +363,51 @@
           "expected": [-2, 0]
         }
       ]
+    },
+    {
+      "description": "Operations between real numbers (int/float) and complex numbers",
```

```suggestion
      "description": "Operations between real numbers and complex numbers",
```

Number types vary between languages, so I think it's best to stick to `real numbers` here and let the tracks map that to whatever type fits their language.